solidity/hardhat.config.ts

```diff
@@ -206,6 +206,7 @@ const config: HardhatUserConfig = {
     disambiguatePaths: false,
     runOnCompile: true,
     strict: true,
+    except: ["BridgeStub$"],
```

Comment:
In #394 I went ahead and added `except: ["^contracts/test"],` but I think this version with only `BridgeStub$` is better. cc @dimpar